Massonus/HW11/task_1.py

```diff
@@ -0,0 +1,25 @@
+class AgeError(Exception):
```

Comment:
good job
but look at this code which works the same, but is easier to read and maintain
class AgeError(Exception):
    def __init__(self, message):
        super().__init__(message)


def check_age(age):
    try:
        age = int(age)
        if age <= 0:
            raise AgeError('Negative numbers are unacceptable')

        return f"Alright, you are {age}. And this is an {'even' if age % 2 == 0 else 'odd'} number."

    except ValueError:
        return f"I don't think that your age is '{age}'"
    except AgeError as error:
        return f"Oops, we have an error here: {error}. Please try again."


age_input = input("Enter your age: ")
print(check_age(age_input))